src/Plugins/ComplexCore/src/ComplexCore/Filters/Algorithms/FindArrayStatistics.cpp

```diff
@@ -655,7 +660,8 @@ void FindStatistics(const DataArray<T>& source, const Int32Array* featureIds, co
     auto* modalBinsArrayPtr = dynamic_cast<NeighborList<float32>*>(arrays[11]);
     auto* featureHasDataPtr = dynamic_cast<BoolArray*>(arrays[12]);
 
-#ifdef COMPLEX_ENABLE_MULTICORE
+// #ifdef COMPLEX_ENABLE_MULTICORE
+#if 0
```

Comment:
Check if parallelization disabled is intended.

Reply:
@mmarineBlueQuartz Can we validate that this parallelization has to be disabled?